code/modules/mob/living/carbon/xenomorph/mutators/behavior_delegate.dm

```diff
@@ -65,6 +65,9 @@
 
 /datum/behavior_delegate/proc/melee_attack_additional_effects_self()
 	SEND_SIGNAL(bound_xeno, COMSIG_XENO_SLASH_ADDITIONAL_EFFECTS_SELF)
+	var/datum/action/xeno_action/onclick/xenohide/hide = get_xeno_action_by_type(bound_xeno, /datum/action/xeno_action/onclick/xenohide)
```

Comment:
This feels like it should be in specific behavior delegates for xenos with hide rather than on the base behavior delegate

Reply:
Implemented, hellhound and runner given new behavior datums to allow this.